dask/bytes/tests/test_s3.py

```diff
@@ -78,7 +78,7 @@ def test_get_s3():
 
 def test_write_bytes(s3):
     paths = ['s3://' + test_bucket_name + '/more/' + f for f in files]
-    values = list(files.values())
+    values = [delayed(v) for v in files.values()]
```

Comment:
Written values are required to be dasky things, not raw data. Could optionally check for this in the function, but I think this is better.